src/main/java/core/basesyntax/models/Square.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class Square implements Figure {
+    private final String color;
+    private final double side;
+
+    public Square(String color, double side) {
+        this.color = color;
+        this.side = side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Color: " + color

```

Comment:
The `draw()` method should output the message in the following format: 'Figure: square, area: size sq.units, side: sideLength, color: color'. Please adjust the output format accordingly.